share/functions/__fish_config_interactive.fish

```diff
@@ -20,16 +20,16 @@ function __fish_config_interactive -d "Initializations that should be performed
     set -g __fish_active_key_bindings
 
     if not set -q fish_greeting
-        set -l line1 (_ 'Welcome to fish, the friendly interactive shell')
+        set -l line1 'Welcome to fish, the friendly interactive shell'
```

Comment:
No, these need gettext.

It's only the *completions* that don't (and hence also functions that are only used in completions).